libs/state/docs/api/operators/interfaces.md

```diff
@@ -0,0 +1,36 @@
+# CompareFn
+
+The function which is used by `distinctUntilSomeChanged` to determine if changes are distinct or not.
```

Comment:
also `selectSlices` should get mentioned

Reply:
done